src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+@import 'variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: grid;
+  grid-template-columns: repeat(7, $day-size);
+  gap: $day-gap;
+  box-sizing: border-box;
+  padding: $calendar-padding;
+  width: calc(
+    (7 * $day-size) + (6 * $day-gap) + (2 * $calendar-padding)
+  );
+}
+
+.calendar__day {
+  position: relative;
+  width: $day-size;
+  height: $day-size;
+  box-sizing: border-box;
+  border: $day-border;
+  background-color: $day-color;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &::before {
+    content: '';
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
```

Comment:
It's a good practice to always specify a fallback font for maximum compatibility. You've specified Arial and Helvetica, but you should also specify a generic font family. For example, if Arial and Helvetica are not available, the browser can use any sans-serif font installed on the user's computer.